modules/openapi-generator/src/main/resources/typescript-fetch/package.mustache

```diff
@@ -1,32 +0,0 @@
-{
```

Comment:
I think we should still have a `package.json` and `tsconfig.json`

Reply:
@TiFu I'm happy to put these back in if required. I removed them because i've found that there's a general level of simplicity gained from just having a single file output. But perhaps with [typescript project references](https://blogs.msdn.microsoft.com/typescript/2018/07/12/announcing-typescript-3-0-rc/#project-references) coming in TS 3.0 it might be worth while to have.

Reply:
`package.json` just makes it simpler to include in packages just as a dependency.

`tsconfig.json` should definitely be included, e.g. specify EcmaScritp version.